tasks/eth/fp-recovery/004-recover-bonds/README.md

```diff
@@ -0,0 +1,69 @@
+# Recover Bonds
+
+Status: CONTINGENCY TASK, SIGN AS NEEDED
+
+## Objective
+
+This task executes a `recover` operation of bonds in the `DelayedWETHProxy` on behalf of the owner. This facilitates the recovery of bonds locked in bricked dispute games so they may be redistributed to its participants. Bond recovery should only be used to recover _unclaimed credits_ of fault dispute games that cannot be reclaimed by players during a superchain pause.
+
+### Tx: Recover Locked WETH
+
+Executes the `recover` call to the `DelayedWETHProxy` contract.
+
+**Function Signature:** `recover(uint256)`
+
+**To:** `<user-input>`
+
+**Value:** `0 WEI`
+
+**Raw Input Data:** `0x0ca35682<-----------------------------wad-value------------------------>`
+
+#### Inputs
+
+- `_wad`: a `uint256` of the amount of funds to recover (generated by running `just generate-input <delayed-weth-address>`)
+
+## Preparing the Operation
+
+1. Locate the address of the `DelayedWETHProxy` contract that the owner wishes to recover funds from.
```

Comment:
To find this repository we should likely add the superchain registry link associate to this contract.

Reply:
Some discussion on this superchain registry issue, of course feel free to chime in with any thoughts  https://github.com/ethereum-optimism/superchain-registry/issues/604

Reply:
Only storing the `DisputeGameFactory` in the superchain registry makes sense to me, all the other contracts are derivable from it